### ISSUE TITLE
Missing image in /docs/index.md

### ISSUE THREAD

**Issue by nczitzk** [foambubble/foam#986]
### Describe the bug

When I was going through the documents, I found a missing image with *"Foam kitchen sink, showing a few of the key features"* as its alt text in the [How do I use Foam?](https://foambubble.github.io/foam/#how-do-i-use-foam) chapter, which links to https://foambubble.github.io/foam/assets/images/foam-features-dark-mode-demo.png

### Steps to Reproduce the Bug or Issue

Go to [How do I use Foam?](https://foambubble.github.io/foam/#how-do-i-use-foam)

### Expected behavior

Expect to see the "Foam kitchen sink" image 😃

### Operating System Version

An issue related with docs and no OS involved.

### Visual Studio Code Version

An issue related with docs and no VS Code involved.

**Comment by riccardoferretti** [comment]
Thanks, I have removed the reference now
